pkg/artifact/registry.go

```diff
@@ -127,7 +127,15 @@ func (a registry) fetchRegistryData(launchableID launch.LaunchableID, version la
 	var registryResponse RegistryResponse
 	err = json.Unmarshal(respBytes, &registryResponse)
 	if err != nil {
-		return nil, auth.VerificationData{}, util.Errorf("Could not unmarshal JSON response from artifact registry: %s", err)
+		l := len(respBytes)
```

Comment:
Yeah the idea is mostly to abstract the code to extract the message. Fair point about it being specifically useful for HTTP responses. Cool with not extracting it.